### DISCUSSION TITLE
Not working

### DISCUSSION THREAD

**Discussion by inFqm** [K4ryuu/K4-System#180] [Q&A] [answered]
I'm going mad. CSS is well installed and plugin too. But it wont start.
**plugins logs** : `[EROR] plugin:K4-System An error occurred while loading player cache: Unknown column 'lastseen' in 'field list'`
clantags do not work and i've got this message each round, whatever the value, whatever the number of humans on the server :

![20240514003547_1](https://github.com/K4ryuu/K4-System/assets/153912477/77c96093-08be-4e33-964c-48f7ad14446e)

Please help.

**Comment by K4ryuu** [comment, marked as answer]
You have updated the plugin's version, but you didn't update your MySQL tables. You should go to releases tab and search for the version you switched from. Open up your MySQL and scroll through to the version you changed to and run all MySQL update queries you see.

Or the easier way: Use this only if the data is not important. Remove your current tables and let the plugin generate new ones.
